src/Xamarin.Android.Build.Tasks/Tasks/BuildAppBundle.cs

```diff
@@ -63,9 +63,11 @@ public class BuildAppBundle : BundleTool
 		public string BaseZip { get; set; }
 
 		public string CustomBuildConfigFile { get; set; }
-		
+
 		public string [] Modules { get; set; }
 
+		public string [] MetaDataFiles { get; set; }
```

Comment:
Convention question: should this be an `ITaskItem[]` and not a string, as it's coming from an item group?

Reply:
It can be.... but the other parts of the Task use `string []` so I either go with what is already there or fix up the other code?